src/analysisd/decoders/security_configuration_assessment.c

```diff
@@ -1544,7 +1547,9 @@ static void FillScanInfo(Eventinfo *lf,cJSON *scan_id,cJSON *name,cJSON *descrip
             sprintf(value, "%d", pass->valueint);
         } else if (pass->valuedouble >= 0) {
             sprintf(value, "%lf", pass->valuedouble);
-        } 
+        } else {
+            return;
```

Comment:
This is an unexpected condition: invalid value. Please print a debugging log here.